submissions/lawalNurudeen/html-forms/contact-form.html

```diff
@@ -1,99 +1,137 @@
+              alt=""
+            />
+            <img
+              src="../css/images-and-icons/social-icons/gmail logo.png"
+              alt=""
+            />
+            <img
+              src="../css/images-and-icons/social-icons/youtube logo.png"
+              alt=""
+            />
+          </div>
         </div>
+      </div>
+      <div class="contact-form">
+        <div class="form-heading">
+          <img src="/images-and-icons/social-icons/ic_email_24px.png" alt="" />
+          <p><strong>Or better still send us a message</strong></p>
+        </div>
+        <form>
+          <input type="text" placeholder="Full Name" />
+          <input type="text" placeholder="Email Address" />
+          <input type="text" placeholder="Subject" />
+          <label for="message">Message</label>
+          <textarea name="message" id="message" rows="10"></textarea>
+          <div id="contact-form-button">
+            <button>SEND</button>
+          </div>
+        </form>
+      </div>
     </div>
 
     <footer>
-        <div class="footer-links">
-            <ul class="nav-links">
-                <li><a href="#">Contact Us</a></li>
-                <li><a href="#">About Us</a></li>
-                <li><a href="#">Terms & Conditions</a></li>
-            </ul>
+      <div class="footer-links">
+        <ul class="nav-links">
+          <li><a href="#">Contact Us</a></li>
+          <li><a href="#">About Us</a></li>
+          <li><a href="#">Terms & Conditions</a></li>
+        </ul>
+
+        <div class="newsletter-sociallinks">
+          <div class="newsletter">
+            <p>Sign up for our Newsletter</p>
+            <form>
+              <input type="text" placeholder="Enter your email address" />
+              <button type="submit">SUBMIT</button>
+            </form>
+          </div>
 
-            <div class="newsletter-sociallinks">
-                <div class="newsletter">
-                    <p>Sign up for our Newsletter</p>
-                    <form>
-                        <input type="text" placeholder="Enter your email address">
-                        <button>SUBMIT</button>
-                    </form>
-                </div>
-        
-                <div class="social-links">
-                    <p>CONNECT WITH US</p>
-                    <div class="social-icons">
-                        <img src="../css/images-and-icons/social-icons/facebook logo.png" alt="">
-                        <img src="../css/images-and-icons/social-icons/twiiter logo.png" alt="">
-                        <img src="../css/images-and-icons/social-icons/gmail logo.png" alt="">
-                        <img src="../css/images-and-icons/social-icons/youtube logo.png" alt="">
-                    </div>
-                </div>
+          <div class="social-links">
+            <p>CONNECT WITH US</p>
+            <div class="social-icons">
+              <img
```

Comment:
Revert this change. All these image tags can take a single line, just as it was before

Reply:
@adaugochi those realignments were done by the PRETTIER extension I have installed on my VSCODE...

Reply:
@adaugochi The changes have been made regardless